grudge/op.py

```diff
@@ -224,8 +229,10 @@ def local_d_dx(dcoll: DiscretizationCollection, xyz_axis, vec):
 
     :arg xyz_axis: an integer indicating the axis along which the derivative
         is taken.
-    :arg vec: a :class:`~meshmode.dof_array.DOFArray`.
-    :returns: a :class:`~meshmode.dof_array.DOFArray`\ s.
+    :arg vec: a :class:`~meshmode.dof_array.DOFArray` or an
+        :class:`~arraycontext.container.ArrayContainer` of them.
+    :returns: a :class:`~meshmode.dof_array.DOFArray` or an
+        :class:`~arraycontext.container.ArrayContainer` of them.
```

Comment:
Are you sure this function supports array containers?

Reply:
It definitely does now :smile: https://github.com/inducer/grudge/pull/154/commits/eeb80eb3554ebd3d80be6e68ca71d38d92b931aa